BEST_PRACTICES.md

```diff
@@ -533,12 +533,16 @@ Always used named parameters, and never unamed ordered parameters (`$0`). It's h
 
 **Preferred:**
 ```swift
-array.map { i in return i ** i }
+let values = [2.0,4.0,5.0,7.0]
+let squares = values.map { element in element * element}
```

Comment:
I would go for `value in value * value` haha, damn, `$0 * $0` is much better for this specific case.